plasmapy/physics/parameters.py

```diff
@@ -474,9 +482,13 @@ def kappa_thermal_speed(T, kappa, particle="e-", method="most_probable"):
         The particle temperature is invalid or particle cannot be used to
         identify an isotope or particle.
 
-    UserWarning
+    Warns
+    -----
+    RelativityWarning
         If the particle thermal speed exceeds 10% of the speed of light, or
-        if units are not provided and SI units are assumed.
+
+    ~astropy.units.UnitsWarning
+        If units are not provided and SI units are assumed.
```

Comment:
I think you want to delete the "and" and add a comma after "provided"

Reply:
Yeah, I just quickly adapted the existing description.